mxcubeweb/core/adapter/__init__.py

```diff
@@ -0,0 +1,36 @@
+import importlib
```

Comment:
`__init_subclass__` requires that the the adapters are loaded otherwise It wont get called. So I added this sort of "boiler plate" module-load code to import all modules with the `_adapter` suffix and classes with the `Adapter` suffix